src/main/java/ch/njol/skript/aliases/Aliases.java

```diff
@@ -231,95 +234,132 @@ public static ItemType parseAlias(final String s) {
 	private final static RegexMessage p_every = new RegexMessage("aliases.every", "", " (.+)", Pattern.CASE_INSENSITIVE);
 	private final static RegexMessage p_of_every = new RegexMessage("aliases.of every", "(\\d+) ", " (.+)", Pattern.CASE_INSENSITIVE);
 	private final static RegexMessage p_of = new RegexMessage("aliases.of", "(\\d+) (?:", " )?(.+)", Pattern.CASE_INSENSITIVE);
-	
+	private final static RegexMessage p_named = new RegexMessage("aliases.named", "(.+) ", " \"(.+)\"", Pattern.CASE_INSENSITIVE);
+	private final static RegexMessage p_with_lore = new RegexMessage("aliases.with lore", "(.+) ", " (\".+\")", Pattern.CASE_INSENSITIVE);
+
+	private final static Pattern ENCHANTMENTS_PATTERN = Pattern.compile("\\s*(,|" + Pattern.quote(Language.get("and")) + ")\\s*");
+	private final static Pattern MULTIPLE_VALUE_SPLIT_PATTERN = Pattern.compile("(, ?| " + Pattern.quote(Language.get("and")) + " )");
+
+	private final static Message m_named = new Message("aliases.named");
+	private final static Message m_with_lore = new Message("aliases.with lore");
+
 	/**
 	 * Parses an ItemType.
 	 * <p>
 	 * Prints errors.
 	 * 
-	 * @param s
+	 * @param value
 	 * @return The parsed ItemType or null if the input is invalid.
 	 */
 	@Nullable
-	public static ItemType parseItemType(String s) {
-		if (s.isEmpty())
+	public static ItemType parseItemType(String value) {
+		if (value.isEmpty())
 			return null;
-		s = "" + s.trim();
-		
-		final ItemType t = new ItemType();
+		value = "" + value.trim();
```

Comment:
```suggestion
		value = value.trim();
```
These should be removable. IIRC they are leftover from old null analysis